server.py

```diff
@@ -38,14 +38,32 @@ def index():
 def list_products():
     category        = request.args.get('category')
     discontinued    = str2bool(request.args.get('discontinued'))
-    results     = []
+    min_Price       = request.args.get('minPrice')
```

Comment:
Just to clarify what was said in class, the recommendation for using a dash was for the URL up to the query string `?` but not including the query string. The query string is probably going to match whatever language you are using so a python program would want `min_price` and `max_price` while a Java program would probably want `minPrice` and `maxPrice`. Since the query string is really data that you are sending to the URL, it's ok for the data to be specified in a format that fits the underlying language.

In the example above, `min_Price` doesn't follow either of those two because it's not camel case nor snake case but rather a combination of both. For a python backend I would expect `min_price` to be more a natural fit.